### ISSUE TITLE
1.9.4 Multi-Player problems

### ISSUE THREAD

**Issue by WhiteFire-Sondergaard** [CyclopsMC/ColossalChests#33]
I have two issues that only happen in multi-player. If I copy the world and run it single player I can not reproduce them.

The first is if you fill a 5x5x5 Diamond chest, re-approaching the chest (say entering the dimension in sight of it) causes: Tried to read NBT tag that was too big; tried to allocate 2097171 bytes where max allowed is: 2097152 causing the client to lose connection instantly.

The second issue is when breaking a chest wall, anything other than the wooden one gives me something different back. Copper walls give me Iron walls, Iron walls give me the next one, after that they return a block with no texture and if you mouse over it in your inventory, instant crash.

Forge 12.17.0.1976, colossal chests 1.3.7

**Comment by rubensworks** [comment]
For your first issue, I anticipated that this might happen. That is why I added the `maxSlotsPerPacket` option in the config file. Try _decreasing_ this value, and see if this your problem. Let me know which value worked for you, and I'll use that as default value for the next update.

For the second problem, I tried this in both SSP and SMP, but I can not seem to reproduce it. I tried breaking all blocks (wall, core, interface) for all variants, and they drop just fine for me. Are you doing anything special? At what version of the Colossal Chests mod were the chests placed originally?

Also, try to always include full crash logs (pastebin), this makes it a lot easier for me to pinpoint the issue.

Thanks for reporting this! ;-) (Next time, try to open separate issues for every problem, makes it a little easier for me)
